src/api/app/views/webui2/webui/patchinfo/_side_elements.html.haml

```diff
@@ -0,0 +1,32 @@
+%ul.list-unstyled
+  - if patchinfo.block
```

Comment:
I agree. I would take this even further, we don't need a collapse for the block reason.

Reply:
okey, I will remove the collapse.

Reply:
@dmarcoux, you prefer to have the `blocked reason`  inside the `title` of the blocked badge? Maybe is too hidden

Reply:
The reason why a patchinfo update is blocked is one of the most important information. It should be prominent.

This isn't perfect, but here's a proposal. The spacing and positioning could be improved, but I think it would be already a step in the right direction:

![proposal](https://user-images.githubusercontent.com/1102934/56295941-e1217680-612d-11e9-9bfe-582ba481f063.png)

This is the HTML:

```
<div class="mt-1">
  <span class="badge badge-blocked" title="Release of this patchinfo is blocked">blocked</span>
  <span class="badge badge-category security" title="Category of this patchinfo">security</span>
  <span class="badge badge-rating moderate" title="Rating of this patchinfo">moderate</span>
  
  <p class="text-danger m-0">This is causing issues! Wait!!! </p>
</div>
```

I'm not convinced though, because the top box with the h3, the badges, side information, bottom actions, summary and description is getting convoluted.

Reply:
and this option?
![Screenshot_2019-04-17 Open Build Service(2)](https://user-images.githubusercontent.com/1212806/56296938-b3d5c800-612f-11e9-9add-8744538473fe.png)

Reply:
hm I prefer the version from @dmarcoux , maybe somebody from maintenance could giv us some feedback here?

Reply:
Another proposal, using an alert message (which isn't dismissable)

![another-proposal](https://user-images.githubusercontent.com/1102934/56297971-a4577e80-6131-11e9-942d-e13bc6cc0196.png)

Reply:
I like the alert message, it is consistent with the rest of the app.

Reply:
I think the reason in the flash message is bad, because the reason can become really lengthy and verbose

Reply:
If the reason is longer, it would still fit in the alert, so it wouldn't be an issue.

Reply:
talking with our maintenance team, they prefer the first option presented by @dmarcoux , @hennevogel . Not sure if a multiline flash message is the right way to do it.

Reply:
Even if the maintenance team prefers my first proposal, I would still favor my second proposal since it's consistent with the rest of the Bootstrap UI (so I agree with @Ana06 here). On top of the consistency, the second proposal also clearly displays the block reason and makes the top box (with the h3, summary, etc..) more readable since there's isn't as much information in it.

Alerts with multiline content are fine, see the examples from [Bootstrap](https://getbootstrap.com/docs/4.3/components/alerts/#additional-content).

Reply:
@dmarcoux sorry, but i failed to find where we consistently show a  desired state set by the user as a sticky error flash message? I was looking in other sections we either have a legend, or a list with error icons or a table showing "broken".. could you point me where we have something similar with the "use setting the patchinfo to stop" and  "show a sticky error message" (possible multiline including references to external bugs) ?

Reply:
I think the basic misunderstanding is that @dmarcoux expected the block reason to be a problem with the patch. But in reality it's always just a halting state.

The patch itself is fine and tested, but the team waits a little longer before releasing to make it worth the extra overhead.

So an error flash is wrong, but I still like the idea of using a flash for this - as it's a rather important info.

Reply:
(And IMO this is a pure misuse of this field, it should be a review on the release request, but then you have the problem that they appear as Tasks on your page as OBS doesn't allow to ignore reviews as tasks)

Reply:
If we don't use an alert for this, it's just going to be hidden among the text of the top box. Since it's a state and it is important, I would prefer to use the alert. If you think it shouldn't be an error, than we can change it to a warning or even info.

Regarding other cases, I was thinking about the build results where we also use it to display the state in a sticky alert when it's building. We do it because it's an important information and shouldn't be overlooked by users.

Reply:
> If we don't use an alert for this, it's just going to be hidden among the text of the top box.

When I checked this view in the reviewapp I first didn't recognize the flash error. It took me a bit to realize that this is probably because my brain is trained that this area is only important when I just fired an action / saved some data.
The second reason might be that it's at the edge of the viewport (I also scrolled down a little bit).
Might be that users would adjust to this, but I still think using a flash like appearance has to many downsides.

I think the core problem is that we want to give this some emphasize, but because we use many icons and colors it becomes tricky to do this.